keras_cv/layers/patch_merging.py

```diff
@@ -0,0 +1,66 @@
+"""
+Code copied and modified from
+https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/swin_transformer.py
+https://github.com/sayakpaul/swin-transformers-tf/blob/main/swins/layers/patch_merging.py
+"""
+
+from functools import partial
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+class PatchMerging(layers.Layer):
+    """Patch Merging Layer.
+    Args:
+        input_resolution (tuple[int]): Resolution of input feature.
+        dim (int): Number of input channels.
+    """
+
+    def __init__(
+        self,
+        input_resolution,
+        dim,
+        out_dim=None,
+        norm_layer=partial(layers.LayerNormalization, epsilon=1e-5),
```

Comment:
instead just default this to None, and describe the default value in the docstring.

```
norm_layer=None,
...
norm_layer = norm_layer or partial(layers.LayerNormalization, epsilon=1e-5)
```

Reply:
And for LayerNorm, I guess the default gold standard for Transformers is 1e-6.